driver/pace/driver/initialization.py

```diff
@@ -136,15 +136,25 @@ class RestartConfig(Initializer):
 
     path: str = "."
     start_time: datetime = datetime(2000, 1, 1)
+    fortran_data: bool = False
 
     def get_driver_state(
         self,
         quantity_factory: pace.util.QuantityFactory,
         communicator: pace.util.CubedSphereCommunicator,
     ) -> DriverState:
         state = _restart_driver_state(
-            self.path, communicator.rank, quantity_factory, communicator
+            self.path,
+            communicator.rank,
+            quantity_factory,
+            communicator,
+            self.fortran_data,
         )
+
+        # TODO
+        # follow what fortran does with restart data after reading it
+        # should eliminate small differences between restart input and
+        # serialized test data
```

Comment:
Is this TODO still relevant? If the directory is a pace restart directory, do we even want to do that?

Reply:
Yes, I think so -- but it now only exists in the fortran restart config